Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -121,6 +121,16 @@ void initEntityPropertyEditorResources()
     Q_INIT_RESOURCE(OverrideResources);
 }
 
+static constexpr const char* enableDPECVarName = "ed_enableDPEInspector";
+
+AZ_CVAR(
+    bool,
+    ed_enableDPEInspector,
+    false,
+    nullptr,
+    AZ::ConsoleFunctorFlags::DontReplicate | AZ::ConsoleFunctorFlags::DontDuplicate,
+    "If set, enables experimental Document Property Editor support for the Entity Inspector");
+
```

Comment:
These should be in the AzToolsFramework below.

Reply:
done